src/gfn/utils/modules.py

```diff
@@ -87,3 +87,19 @@ def __call__(self, preprocessed_states: InputTensor) -> OutputTensor:
         assert preprocessed_states.dtype == torch.long
         outputs = self.table[preprocessed_states.squeeze(-1)]
         return outputs
+
+
+class Uniform(nn.Module):
+    def __init__(self, output_dim: int) -> None:
+        """Implements a zero function approximator, i.e. a function that always outputs 0.
```

Comment:
The reason I had removed it is that it doesn't make sense to have a uniform distribution (over parents or children) for a generic environment. But I agree it would be useful to have it for discrete environments. Could you make a better docstring, mentioning e.g. that output_dim is usually n_actions - 1 in discrete environments (representing the maximum number of parents)

Reply:
@vict0rsch ?